methods/methods_release2.md

```diff
@@ -93,49 +95,262 @@ Demultiplexing, feature-table generation, fragment insertion, and taxonomic prof
+done`
 
-#### Taxonomic profiling
-* Process: Feature-classifier sklearn 
-* Parameters: Using the SILVA 138.1 release as a reference
+#### Denoising
+`deblur workflow \
+  --seqs-fp emp500_18s_demux_trimmed_merged_files/ \
+  --output-dir emp500_18s_deblur/ \
+  --trim-length 90 \
+  --threads-per-sample 0 \
+  --overwrite`
 
-#### OTU clustering
-* Process: Closed-reference OTU picking 
-* Parameters: 97% sequence similarity threshold, using the SILVA 119 release as a reference
+Import denoised sequences and feature-table into QIIME 2
+
+`qiime tools import \
+  --input-path emp500_18s_deblur/all.biom \
+  --type 'FeatureTable[Frequency]' \
+  --input-format BIOMV210Format \
+  --output-path emp500_18s_deblur/all_biom.qza`
+
+`qiime tools import \
+  --input-path emp500_18s_deblur/all.seqs.fa \
+  --output-path emp500_18s_deblur/all_seqs.qza \
+  --type 'FeatureData[Sequence]'`
+  
+At this point denoised feature-tables and sequences from each sequencing lane were merged.
 
+#### Merging tables:
+`qiime feature-table merge \
+  --i-tables emp500_18s_deblur_prep_8694_biom.qza \
+  --i-tables emp500_18s_deblur_prep_8753_biom.qza \
+  --i-tables emp500_18s_deblur_prep_8754_biom.qza \
+  --i-tables emp500_18s_deblur_prep_9950_biom.qza \
+  --i-tables emp500_18s_deblur_prep_9951_biom.qza \
+  --i-tables emp500_18s_deblur_prep_9952_biom.qza \
+  --i-tables emp500_18s_deblur_prep_9953_biom.qza \
+  --i-tables emp500_18s_deblur_prep_9954_biom.qza \
+  --i-tables emp500_18s_deblur_prep_9955_biom.qza \
+  --p-overlap-method 'sum' \
+  --o-merged-table emp500_18s_deblur_merged_biom.qza`
+
+`qiime feature-table merge-seqs \
+  --i-data emp500_18s_deblur_prep_8694_seqs.qza \
+  --i-data emp500_18s_deblur_prep_8753_seqs.qza \
+  --i-data emp500_18s_deblur_prep_8754_seqs.qza \
+  --i-data emp500_18s_deblur_prep_9950_seqs.qza \
+  --i-data emp500_18s_deblur_prep_9951_seqs.qza \
+  --i-data emp500_18s_deblur_prep_9952_seqs.qza \
+  --i-data emp500_18s_deblur_prep_9953_seqs.qza \
+  --i-data emp500_18s_deblur_prep_9954_seqs.qza \
+  --i-data emp500_18s_deblur_prep_9955_seqs.qza \
+  --o-merged-data emp500_18s_deblur_merged_seqs.qza`
+
+#### Taxonomic profiling:
+`qiime feature-classifier classify-sklearn \
+  --i-reads emp500_18s_deblur_merged_seqs.qza \
+  --i-classifier silva-138-99-nb-classifier.qza \
+  --o-classification emp500_18s_deblur_merged_seqs_taxonomy_silva138.qza`
+
+#### Filtering reads classified as bacteria and archaea
+`qiime feature-table filter-features \
+  --i-table emp500_18s_deblur_merged_biom.qza \
+  --m-metadata-file emp500_18s_deblur_merged_seqs_silva138_feature_metadata.txt \
+  --p-where 'Domain != "Bacteria" AND Domain != "Archaea"' \
+  --o-filtered-table emp500_18s_deblur_merged_biom_euks.qza`
+
+#### Filtering singleton features on a per-sample basis
+`qiime feature-table filter-features \
+  --i-table emp500_18s_deblur_merged_biom_euks.qza \
+  --p-min-samples 2 \
```

Comment:
This isn't a singleton filter, but a minimum number of samples. Usually singleton filters drop features with a read count of 1?

Reply:
Thanks! Singletons can be defined as you described but also on a per-sample basis and the latter is what is described here (see line 207). In general a per-read singleton filter is not as conservative as a per-sample one, as the latter requires the feature to be observed in >1 sample vs. just being found twice in a single sample.